core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -732,27 +745,69 @@ portfolio.pending.tradePeriodInfo=After the first blockchain confirmation, the t
 portfolio.pending.tradePeriodWarning=If the period is exceeded both traders can open a dispute.
 portfolio.pending.tradeNotCompleted=Trade not completed in time (until {0})
 portfolio.pending.tradeProcess=Trade process
-portfolio.pending.openAgainDispute.msg=If you are not sure that the message to the arbitrator arrived (e.g. if you did not get a response after 1 day) feel free to open a dispute again.
+portfolio.pending.openAgainDispute.msg=If you are not sure that the message to the mediator or arbitrator arrived \
```

Comment:
Should we mention the shortcut here (Cmd/Ctrl+o)?

Also should we mention the forum URL (https://bisq.community)?

Suggestion:

```
portfolio.pending.openAgainDispute.msg=If you are not sure that the message to the mediator or arbitrator arrived / (e.g. if you did not get a response after 1 day) feel free to open a dispute again with Cmd/Ctrl+o. You can also ask for additional help on the Bisq forum at https://bisq.community.
```

Reply:
Yes, agree. Can you make another suggestion so I can commit here?

Reply:
Applied your text in next commit.